ci-operator/step-registry/configure-registry-storage/nutanix/configure-registry-storage-nutanix-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+echo "$(date -u --rfc-3339=seconds) - Configuring image registry with emptyDir..."
+oc patch configs.imageregistry.operator.openshift.io cluster --type merge --patch '{"spec":{"managementState":"Managed","storage":{"emptyDir":{}}}}'
+echo "$(date -u --rfc-3339=seconds) - Wait for the imageregistry operator to see that it has work to do..."
+sleep 30
+echo "$(date -u --rfc-3339=seconds) - Wait for the imageregistry operator to go available..."
+oc wait --all --for=condition=Available=True clusteroperators.config.openshift.io --timeout=10m
```

Comment:
This is waiting for all the operators get ready?

Reply:
The patch operation will also affect other co rollout, image registry is already available after patch, but other co are still in progress
![image](https://github.com/openshift/release/assets/12492932/91bc15ef-8362-4d12-beca-460ea491caaa)